Sources/TuistKit/GraphMappers/WorkspaceMapperProvider.swift

```diff
@@ -0,0 +1,27 @@
+import Foundation
+import TuistCore
+
+protocol WorkspaceMapperProviding {
+    func mapper(config: Config) -> WorkspaceMapping
+}
+
+final class WorkspaceMapperProvider: WorkspaceMapperProviding {
+    private let projectMapperProvider: ProjectMapperProviding
+    init(projectMapperProvider: ProjectMapperProviding = ProjectMapperProvider()) {
+        self.projectMapperProvider = projectMapperProvider
+    }
+
+    func mapper(config: Config) -> WorkspaceMapping {
+        var mappers: [WorkspaceMapping] = []
+
+        mappers.append(
+            ProjectWorkspaceMapper(mapper: projectMapperProvider.mapper(config: config))
```

Comment:
We're able to integrate and compose any existing `Project` mappers via `ProjectWorkspaceMapper`

Reply:
also super nice 👏